File: app/src/main/java/com/digicoffer/lauditor/Dashboard/DahboardModels/PracticeHeadModels/ApproxRevenueModel.java
package com.digicoffer.lauditor.Dashboard.DahboardModels.PracticeHeadModels;

import org.json.JSONObject;

public class ApproxRevenueModel {
    String approxRevenue;
    String CurrencySymbol="";
    String CurrencyCode="";

    public ApproxRevenueModel(String approxRevenue,String CurrencySymbol, String CurrencyCode) {
        this.approxRevenue = approxRevenue;
        this.CurrencySymbol=CurrencySymbol;
        this.CurrencyCode=CurrencyCode;
    }

    public String getCurrencyCode() {
        return CurrencyCode;
    }

    public String getCurrencySymbol() {
        return CurrencySymbol;
    }

    public String getApproxRevenue() {
        return approxRevenue;
    }

    public void setApproxRevenue(String approxRevenue) {
        this.approxRevenue = approxRevenue;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Dashboard/DahboardModels/PracticeHeadModels/ApproxRevenueModel.kt
package com.digicoffer.lauditor.Dashboard.DahboardModels.PracticeHeadModels

class ApproxRevenueModel(
    var approxRevenue: String?,
    @get:JvmName("getCurrencySymbol") val currencySymbol: String?,
    @get:JvmName("getCurrencyCode") val currencyCode: String?
)
